datacompy/core.py

```diff
@@ -749,6 +749,9 @@ def render(filename: str, *fields: Union[int, float, str]) -> str:
         return file_open.read().format(*fields)
 
 
+import pandas as pd
```

Comment:
These should be placed at the top of the file as it doesn't conform to our code quality guidelines.

Reply:
Hi @fdosani 
Thanks for highlighting this. We will look into this as to why this happened.

BR,
Team Ana